sqlglot/parser.py

```diff
@@ -739,6 +741,27 @@ def _parse_character_set(self, default=False):
             default=default,
         )
 
+    def _parse_returns(self):
+        value = None
+        is_table = False
+        if self._match(TokenType.TABLE):
+            is_table = True
+            if self._match(TokenType.LT):
+                value = self.expression(
+                    exp.Schema, this="TABLE", expressions=self._parse_csv(self._parse_struct_kwargs)
+                )
+                if not self._match(TokenType.GT):
+                    self.raise_error("Expecting >")
+            else:
+                value = self._parse_schema("TABLE")
```

Comment:
what is this hardcoded TABLE?

Reply:
`TABLE` is a special data type unique (I think?) to UDFs.